static/app/components/feedback/feedbackItem/screenshotsModal.tsx

```diff
@@ -0,0 +1,88 @@
+import {Fragment, useState} from 'react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import type {ModalRenderProps} from 'sentry/actionCreators/modal';
+import ScreenshotPagination from 'sentry/components/events/eventTagsAndScreenshot/screenshot/screenshotPagination';
+import FeedbackScreenshot from 'sentry/components/feedback/feedbackItem/feedbackScreenshot';
+import PanelHeader from 'sentry/components/panels/panelHeader';
+import {t, tct} from 'sentry/locale';
+import type {EventAttachment, Organization, Project} from 'sentry/types';
+
+type Props = ModalRenderProps & {
+  initialIndex: number;
+  organization: Organization;
+  projectSlug: Project['slug'];
+  screenshots: EventAttachment[];
+};
+
+export default function ScreenshotsModal({
+  Body,
+  Header,
+  initialIndex,
+  organization,
+  projectSlug,
+  screenshots,
+}: Props) {
+  // Selected index can be any integer, positive or negative
+  const [selectedIndex, setSelectedIndex] = useState(initialIndex);
```

Comment:
when is selected index negative? :o

Reply:
Starting at index 0, you can click 'prev' and it'll go into negative values, wrapping around list of images.